internal/model/types/file.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	"bytes"
+	"os"
+)
+
+type File struct {
```

Comment:
Since the struct and its members are exported, can you add a description?

(running `make lint` will give similar feedback)